console/src/hardware/task/ontology.tsx

```diff
@@ -77,6 +83,7 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
       <Menu.Item itemKey="delete" startIcon={<Icon.Delete />}>
         Delete
       </Menu.Item>
+      <Ontology.LinkAddressMenuItem />
```

Comment:
This seems like it'd be a better fit for the `Link` module instead of the ontology